src/matcha_ml/infrastructure/modules/seldon/seldon/istio.tf

```diff
@@ -0,0 +1,65 @@
+# Derived from ZenML's stack recipes; source: https://github.com/zenml-io/mlops-stacks/blob/8eb06596bf836d3a3dd2634fbc7f2b5687421811/aws-minimal/seldon/istio.tf
+
+# create a namespace for istio resources
+resource "kubernetes_namespace" "istio_ns" {

```

Comment:
Is istio only used by Seldon? If not, this needs to be in common module

Reply:
It seems it is only used by Seldon